Source/items.cpp

```diff
@@ -2697,7 +2697,11 @@ void CalcPlrItemVals(Player &player, bool loadgfx)
 		int8_t ticksPerFrame;
 		player.getAnimationFramesAndTicksPerFrame(graphic, numberOfFrames, ticksPerFrame);
 		LoadPlrGFX(player, graphic);
-		player.AnimInfo.changeAnimationData(player.AnimationData[static_cast<size_t>(graphic)].spritesForDirection(player._pdir), numberOfFrames, ticksPerFrame);
+		OptionalClxSpriteList sprites;
+		if (!HeadlessMode) {
+			sprites = player.AnimationData[static_cast<size_t>(graphic)].spritesForDirection(player._pdir);
+		}
+		player.AnimInfo.changeAnimationData(sprites, numberOfFrames, ticksPerFrame);

```

Comment:
Oh nice I belive this is also what prevented me from testing a lot of the various game parts in the timedemo